src/interfaces/index.ts

```diff
@@ -188,12 +188,15 @@ export interface FlatFareProductDetails extends BaseProduct {
     productPrice: string;
 }
 
-export interface ProductDetails extends BaseProduct {
+export interface ProductDetail {
```

Comment:
Naming wise, it looks a little strange to call this ProductDetail I think. Does the newer ProductDetails interface need to have that name?

Reply:
Would the type already there not work?

Reply:
can rename but the type there has the sale offer package in there but will rename

Reply:
Using the Product interface now